src/runtime/README.md

```diff
@@ -132,27 +127,13 @@ components, see the documentation for the
 [`kata-log-parser`](https://github.com/kata-containers/tests/tree/main/cmd/log-parser)
 tool.
 
-For runtime logs, see the following sections for the CRI-O and containerd shimv2 based runtimes.
-
-### Kata OCI
-
-The Kata OCI runtime (including when used with CRI-O), provides `--log=` and `--log-format=` options.
-However, the runtime also always logs to the system log (`syslog` or `journald`).
-
-To view runtime log output:
-
-```bash
-$ sudo journalctl -t kata-runtime
-```
-
 ### Kata containerd shimv2
 
 The Kata containerd shimv2 runtime logs through `containerd`, and its logs will be sent
 to wherever the `containerd` logs are directed. However, the
-shimv2 runtime also always logs to the system log (`syslog` or `journald`) under the
-identifier name of `kata`.
+shimv2 runtime also always logs to the system log (`syslog` or `journald`) using the `kata` identifier.
```

Comment:
This requires containerd debug to be enabled. Should we mention that here?

Reply:
Fixed - ptal.